src/cascadia/WindowsTerminal/NonClientIslandWindow.cpp

```diff
@@ -896,6 +905,10 @@ void NonClientIslandWindow::_SetIsBorderless(const bool borderlessEnabled)
         _titlebar.Visibility(_IsTitlebarVisible() ? Visibility::Visible : Visibility::Collapsed);
     }
 
+    // Update the margins when entering/leaving focus mode, so we can prevent
+    // the titlebar from showing through transparent terminal controls
+    _UpdateFrameMargins();

```

Comment:
fwiw the titlebar now appears behind all terminals on resize

Reply:
not just fomo

Reply:
Yea. Couldn't find a way to suppress that. Before the island would paint on top of where the win32 titlebar is, so it wouldn't be visible. Now the island is transparent, so there's nothing to obscure the titlebar. 

We could _maybe_ paint the region that's between the side of the window and the island with some color, then paint over it with `BLACK_BRUSH` once the island resizes, but I'm not sure if that'll work, or if we get the right paint messages to enable that.